apps/datahub-e2e/src/e2e/organization-page.cy.ts

```diff
@@ -0,0 +1,125 @@
+import 'cypress-real-events'
+
+describe('organizations', () => {
+  beforeEach(() => {
+    cy.visit('organization/Barbie%20Inc.')
+
+    // aliases
+    cy.get('gn-ui-navigation-button').as('backButton')
+    cy.get('[data-cy="organizationHeaderName"]').as('organizationHeaderName')
+    cy.get('[data-cy="organizationHeaderWebsiteLink"]').as(
+      'organizationHeaderWebsiteLink'
+    )
+    cy.get('[data-cy="organizationDescription"]').as('organizationDescription')
+    cy.get('gn-ui-max-lines').contains('Read more').as('readMoreButton')
+    cy.get('[data-cy="organizationLogo"]').as('organizationLogo')
+    cy.get('[data-cy="organizationDatasetCount"]').as(
+      'organizationDatasetCount'
+    )
+    cy.get('[data-cy="organizationEmail"]').as('organizationEmail')
+    cy.get('[data-cy="organizationPageLastPublishedDatasets"]').as(
+      'organizationPageLastPublishedDatasets'
+    )
+    cy.get(
+      '[data-cy="organizationDetailsLastPublishedDatasetsSearchAllButton"]'
+    ).as('organizationDetailsLastPublishedDatasetsSearchAllButton')
+  })
+
+  describe('general display', () => {
+    describe('header', () => {
+      describe('back button', () => {
+        beforeEach(() => {
+          // Simulate that we come from the organizations search page
```

Comment:
comments to be removed

Reply:
Why ? It is a usefull comment. I wrote it to explain why I go to the organisations page before actually going to the oraganization page.

Reply:
Because it is not best practice to keep comments IMO. Also, I don't feel like this part of the code is complicated enough for an explanation. Same response for the other comments, it's not hard to understand that the code describes columns, since it's also in the id :slightly_smiling_face:

Reply:
Ok.